base/deprecated.jl

```diff
@@ -440,3 +440,11 @@ export float32_isvalid, float64_isvalid
 @deprecate (&)(x::Char, y::Char)  Char(UInt32(x) & UInt32(y))
 @deprecate (|)(x::Char, y::Char)  Char(UInt32(x) | UInt32(y))
 @deprecate ($)(x::Char, y::Char)  Char(UInt32(x) $ UInt32(y))
+
+# 11241
+
+@deprecate is_valid_char(ch::Union(Unsigned, Integer, Char)) isvalid(Char, ch)
+@deprecate is_valid_ascii(str::Union(Vector{UInt8}, ByteString)) isvalid(ASCIIString, str)
```

Comment:
> boy, you guys are strict... now, how did some of the rather unclear code I’ve found ever get in, if you are always this strict!

The standards have gotten stricter over time for new code, as you've probably noticed. We could do with a few sprints/PR's on cleaning up old code to match the standards we're holding more recent PR's to.